core/commands/partials/vagrant_configurator.rb

```diff
@@ -239,11 +243,16 @@ def up
     run_in_directory(@config.path) do
       store_network_config
       up_results = Workers.map(nodes) { |node| up_node(node) }
-      up_results.each { |up_result| up_result[1].print_to_stdout } if @env.threads_count > 1
+      up_results.each { |up_result| up_result[1].print_to_stdout } if check_threads_and_nodes_count
       return ERROR_RESULT unless up_results.detect { |up_result| !up_result[0] }.nil?
     end
     generate_config_information(Dir.pwd)
     generate_label_information_file
     SUCCESS_RESULT
   end
+
+  # Check threads_count and number of nodes
+  def check_threads_and_nodes_count
```

Comment:
The name of the method describes the operation of the method, not it's intent. If in the future we will modify the logic of the method in some way we will be forced to change the name too. This is a bad practice. Please encapsulate the implementation of the method behind the proper name.

Please use names that describes the result of the method. For current method `use_log_storage?` seems to be much better.